File: TestfileCreator/TestfileCreator.py
import random
import os
import time

def createTestFile():
    if not os.path.exists("./Testfiles/"):
        os.mkdir("./Testfiles")
    os.system("base64 /dev/urandom | head -c 10000000 > ./Testfiles/Testfile00.txt")
    time.sleep(5)
    f = open("./Testfiles/Testfile00.txt")
    text = f.read()
    for i in range(1,50):
        modified = text
        for j in range(5):
            index = random.randint(0,9999999)
            modified = modified[:index]+str(random.randint(0,9))+modified[index+1:]
        path = "./Testfiles/Testfile"+str(i).zfill(2)+".txt"
        w = open(path,'w')
        w.write(modified)
        w.close()
    f.close()

File: TestfileCreator/src/TestFileCreator.java
import java.io.*;
import java.util.concurrent.ThreadLocalRandom;

public class TestFileCreator {

    public static void main(String[] args) {
        File mypath = new File("./Testfiles");
        if (!mypath.exists())
            mypath.mkdir();
        try {
            String cmdStr = "base64 /dev/urandom | head -c 10000000 > ./Testfiles/Testfile00.txt";
            String[] cmd = new String[]{"/bin/bash", "-c", cmdStr};
            Runtime.getRuntime().exec(cmd);
            System.out.println("Success");
        } catch (IOException e) {
            e.printStackTrace();
            System.out.println("error");
        }
        File file = new File("./Testfiles/Testfile00.txt");
        Long filelength = file.length();
        byte[] filecontent = new byte[filelength.intValue()];
        try {
            FileInputStream in = new FileInputStream(file);
            in.read(filecontent);
            in.close();
        } catch (IOException io) {
            io.printStackTrace();
        }
        try {
            String txt = new String(filecontent, "utf-8");
            int num = 1;
            int k = 0;
            while (num < 50) {
                String index = String.format("%2d", num).replace(" ","0");
                PrintWriter writer = new PrintWriter("./Testfiles/Testfile" + index + ".txt");
                StringBuilder str = new StringBuilder(txt);
                while (k < 5){
                    int randomnum = ThreadLocalRandom.current().nextInt(0,81919);
                    str.setCharAt(randomnum,'l');
                    k += 1;
                }
                writer.write(str.toString());
                writer.close();
                num += 1;
                }
            }catch (FileNotFoundException | UnsupportedEncodingException f) {
            f.printStackTrace();
        }
    }
}
